src/LeakFinder.cpp

```diff
@@ -875,7 +875,7 @@ static int MyAllocHook(int nAllocType, void *pvData,
 				printf("********************************************\n");
 				printf("Please wait\n");
 				
-				LeakFinderXmlOutput Output("memdump.xml");
+				LeakFinderXmlOutput Output(L"memdump.xml");
```

Comment:
Instead of hard-coding string width, you should use the `TEXT()` macro and the `LPTSTR` or `LPCTSTR` typedefs for pointers.